engine/platform/sdl/s_sdl.c

```diff
@@ -141,6 +141,7 @@ qboolean SNDDMA_Init( void )
 	Con_Printf( "Using SDL audio driver: %s @ %d Hz\n", SDL_GetCurrentAudioDriver( ), obtained.freq );
 
 	dma.initialized = true;
+	dma.backendName = "SDL";
```

Comment:
For SDL, we probably can write something like "SDL (%s)", where %s is SDL_GetCurrentAudioDriver output.

Reply:
Yes, but then we need create char array for that in s_sdl.c

Reply:
I did it